README.md

```diff
@@ -2,9 +2,7 @@
 
 ### DB Indexing Exercise in PostgreSQL
 
-When connecting to your pg database, connect from this project directory.
-
-To run sql statements from an external file, use the `\i [filepath]` command.
+To run sql statements from an external file, load them into DBeaver and run them from there.
```

Comment:
Same question from PR in other repo. Should we use product agnostic language to not need to do so many copy edits over time.